modules/apps/27-interchain-accounts/host/keeper/msg_server.go

```diff
@@ -20,9 +24,51 @@ type msgServer struct {
 // NewMsgServerImpl returns an implementation of the ICS27 host MsgServer interface
 // for the provided Keeper.
 func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
+	if keeper.queryRouter == nil {
+		panic("query router must not be nil")
+	}
 	return &msgServer{Keeper: keeper}
 }
```

Comment:
ACK this change looks good to me! As long as we put something into the release notes which document that users should use `app.HostKeeper.WithQueryRouter` in app.go, otherwise nodes won't start!